src/components/discover-sheet/ListsSection.js

```diff
@@ -97,12 +97,26 @@ export default function ListSection() {
   const { colors } = useTheme();
   const listData = useMemo(() => DefaultTokenLists[network], [network]);
 
+  const addressSocket = useSelector(
+    ({ explorer: { addressSocket } }) => addressSocket
+  );
+  useEffect(() => {
+    Object.values(listData).forEach(({ tokens }) => {
+      if (addressSocket !== null) {
```

Comment:
Might make sense to move this check outside of the loop

Reply:
kittos